ironfish/src/utils/currency.ts

```diff
@@ -69,6 +69,20 @@ export class CurrencyUtils {
 
     return ore
   }
+
+  static renderBalance(amount: bigint | string, assetIdentifier?: string): string {
```

Comment:
should the asset identifier be a enum type so its not free form text string

Reply:
no, it should be a string.

Reply:
okay but why